apps/web/src/components/HoverCard/HoverCardCommunityInner.tsx

```diff
@@ -138,16 +138,25 @@ export function HoverCardCommunityInner({
     return null;
   }
 
+  const formatEthAddress = (e: string) => {
+    return e.slice(0, 4) + '...' + e.slice(-4);
+  };
```

Comment:
when defining functions within a react component, we want to make sure to wrap it in `useCallback` for performance reasons.

that said, this function is generic enough where it's defined elsewhere. you should be able to do `import { truncateAddress } from '~/shared/utils/wallet'`